web/impact/impact/v1/views/cancel_office_hour_session_view.py

```diff
@@ -26,28 +26,26 @@
 FAIL_HEADER = 'Office hour session could not be canceled'
 
 
-def get_office_hours_list_url(family_slug, program_slug):
+def get_office_hours_url():
     site_url = 'accelerate.masschallenge.org'
-    return 'https://{}/officehours/list/{}/{}'.format(
-        site_url, family_slug, program_slug
-    )
+    return 'https://{}/newofficehour/'.format(site_url)
 
 
 def get_office_hour_shared_context(office_hour, message=None):
-    family_slug = office_hour.program.program_family.url_slug
-    program_slug = office_hour.program.url_slug
     tz = timezone(office_hour.location.timezone or DEFAULT_TIMEZONE)
     date = office_hour.start_date_time.astimezone(tz).strftime('%A, %d %B, %Y')
     start_time = office_hour.start_date_time.astimezone(tz).strftime('%I:%M%p')
     end_time = office_hour.end_date_time.astimezone(tz).strftime('%I:%M%p')
+    program = office_hour.program
+    phone = program.program_family.phone_number if program else ''
```

Comment:
Not a blocker - we can come back to this another day - but office hours do have Locations, which can have phone numbers. Should be we trying to include those? 

@hfreund this seems like a good thought to put aside, but I wanted to call it to your attention